README.md

```diff
@@ -45,36 +45,53 @@ import (
 )
 
 func main() {
-  fdk.Do(myHandler)
+  fdk.Do(new(mydoer))
 }
 
-func myHandler(ctx context.Context, in io.Reader, out io.Writer) error {
+// use a struct to maintain state between invocations
```

Comment:
Should we promote using global state?  I get that they *can* do this and have some temporary global state, but should we promote it?

Reply:
by having our integration as an interface (vs a `func` type) we are expressly promoting them not to have global state, I would think, at least in the sense of package level variables? I mostly followed the go `net/http` conventions, which seem familiar for most go developers and allow a lot of flexibility for not having state / having state if they'd like.

Reply:
i guess maybe putting an explicit example using a struct in the README is unnecessary, was just trying to show it I guess (it's contrived, I agree) -- I'll remove from README